cmake/IgnUtils.cmake

```diff
@@ -209,7 +210,37 @@ macro(gz_find_package PACKAGE_NAME)
 
   #------------------------------------
   # Call find_package with the provided arguments
-  find_package(${${PACKAGE_NAME}_find_package_args})
+
+  # TODO(CH3): Deprecated. Remove on tock.
+  if(${PACKAGE_NAME} MATCHES "^Ign")
+
+    # Deliberately use QUIET since we expect Ign to fail
+    find_package(${${PACKAGE_NAME}_find_package_args} QUIET)
+
+    if(NOT ${PACKAGE_NAME}_FOUND)
+      # Try Gz prepended version instead!
+      string(REGEX REPLACE "^Ign(ition)?" "Gz" PACKAGE_NAME_GZ ${PACKAGE_NAME})
+
+      set(${PACKAGE_NAME_GZ}_find_package_args ${${PACKAGE_NAME}_find_package_args})
+      list(POP_FRONT ${PACKAGE_NAME_GZ}_find_package_args)
+      list(PREPEND ${PACKAGE_NAME_GZ}_find_package_args ${PACKAGE_NAME_GZ})
+
+      find_package(${${PACKAGE_NAME_GZ}_find_package_args})
+      if(${PACKAGE_NAME_GZ}_FOUND)
+
+        message(DEPRECATION "Ign prefixed package name [${PACKAGE_NAME}] is deprecated! Automatically using the Gz prefix instead: [${PACKAGE_NAME_GZ}]")
+        set(PACKAGE_NAME ${PACKAGE_NAME_GZ})
```

Comment:
this change in `PACKAGE_NAME` means there is no longer a matching `${PACKAGE_NAME}_pretty` variable, since that is set just before this code block. This could be fixed by moving that code block to after this `if` logic

Reply:
Ah, nice catch! It seems despite my best efforts I got bamboozled by macro substitution :sweat: 

https://github.com/gazebosim/gz-cmake/pull/273/commits/98aadc506549c33e228051dee7cfeebc35cf3bcf

Upon approval, I'll squash all the non file-migration commits into one so when we rebase merge we'll just have two commits!